src/org/omegat/core/machinetranslators/ApertiumTranslate.java

```diff
@@ -93,8 +105,17 @@ protected String translate(Language sLang, Language tLang, String text) throws E
         String sourceLang = apertiumCode(sLang);
         String targetLang = apertiumCode(tLang);
 
-        String url2 = GT_URL2.replace("#sourceLang#", sourceLang).replace("#targetLang#", targetLang);
-        String url = GT_URL + URLEncoder.encode(trText, "UTF-8") + url2;
+        String server = getCredential(PROPERTY_APERTIUM_SERVER_URL);
```

Comment:
Do you consider the server URL to be a "credential" (requiring secure storage)? I don't think it should be, in which case it should be stored as a normal preference and not with the credentials API.

Reply:
It is not, so I have changed it to a normal preference. Thanks!